guardrails/guardrail_globals.py

```diff
@@ -135,3 +141,36 @@ def dead_code_exclude(self):
         if self.dead_code_ignore:
             cmd = "--exclude %s" % self.dead_code_ignore
         return cmd
+
+    def read_file_to_list(self):
+        """ Function to read ignored pylint files to list """
+        ret_val = None
+        if self.lint_ignore:
+            with open(r"%s" % self.lint_ignore) as file:
+                content = file.readlines()
+            # Remove whitespace characters like `\n` at the end of each line
+            ret_val = [x.strip() for x in content]
+        return ret_val
+
+    @staticmethod
+    def list_to_str(str_list):
+        """ Function to convert ignored pylint files list to string """
+        str1 = ' '.join(str(e) for e in str_list)
```

Comment:
While this is very local - maybe we can name str1 better?

Reply:
agree, it is refactored to a meaningful string name